stacks-common/src/types/mod.rs

```diff
@@ -73,11 +73,12 @@ pub enum StacksEpochId {
     Epoch2_05 = 0x02005,
     Epoch21 = 0x0200a,
     Epoch22 = 0x0200f,
+    Epoch23 = 0x02014,
```

Comment:
Curious what the bit pattern here means?  e.g why wasn't epoch 2.2 `0x0200b`?

Reply:
They're just all +5.